site/get-started.pug

```diff
@@ -0,0 +1,120 @@
+
+              h3.spectrum-Heading.spectrum-Heading--S Adding Spectrum CSS
+              p.spectrum-Body.spectrum-Body--L Inside the #[code.spectrum-Code.spectrum-Code--S &lt;head&gt;] tag, include the stylesheets that contain the design system tokens for global variables, scales, and color themes, as well as the stylesheets for the components themselves:
+
+              -
+                htmlMarkup = util.Prism.highlight(`<head>
+                  <link rel="stylesheet" href="node_modules/@spectrum-css/vars/dist/spectrum-global.css">
+                  <link rel="stylesheet" href="node_modules/@spectrum-css/vars/dist/spectrum-medium.css">
+                  <link rel="stylesheet" href="node_modules/@spectrum-css/vars/dist/spectrum-light.css">
+                  <link rel="stylesheet" href="node_modules/@spectrum-css/page/dist/index-vars.css">
+                  <link rel="stylesheet" href="node_modules/@spectrum-css/button/dist/index-vars.css">
+                </head>`, util.Prism.languages.markup, 'markup');
+              pre.spectrum-CodeBlock
+                code.spectrum-Code.spectrum-Code--S.language-markup!=htmlMarkup
+
+              p.spectrum-Body.spectrum-Body--L Inside the #[code.spectrum-Code.spectrum-Code--S &lt;body&gt;] tag, add the markup for a Spectrum button. Note that the example also includes the CSS class name, #[code.spectrum-Code.spectrum-Code--S spectrum-Button--cta], to use the call to action (CTA) option.
+
+              -
+                htmlMarkup = util.Prism.highlight(`<button class="spectrum-Button spectrum-Button--cta">
+                  <span class="spectrum-Button-label">Button</span>
+                </button>`, util.Prism.languages.markup, 'markup');
+              pre.spectrum-CodeBlock
+                code.spectrum-Code.spectrum-Code--S.language-markup!=htmlMarkup
+
+              p.spectrum-Body.spectrum-Body--L Here’s what your final button.html file should look like:
+
+              -
+                htmlMarkup = util.Prism.highlight(`<html class="spectrum spectrum--light spectrum--medium">
+                  <head>
+                    <link rel="stylesheet" href="node_modules/@spectrum-css/vars/dist/spectrum-global.css">
+                    <link rel="stylesheet" href="node_modules/@spectrum-css/vars/dist/spectrum-medium.css">
+                    <link rel="stylesheet" href="node_modules/@spectrum-css/vars/dist/spectrum-light.css">
+                    <link rel="stylesheet" href="node_modules/@spectrum-css/page/dist/index-vars.css">
+                    <link rel="stylesheet" href="node_modules/@spectrum-css/button/dist/index-vars.css">
+                  </head>
+                  <body>
+                    <button class="spectrum-Button spectrum-Button--cta">
+                      <span class="spectrum-Button-label">Button</span>
+                    </button>
+                  </body>
+                </html>`, util.Prism.languages.markup, 'markup');
```

Comment:
Ok I know this is pretty gross, but its the best way I could come up with. We should definitely change it later.